inference-engine/thirdparty/clDNN/kernel_selector/CMakeLists.txt

```diff
@@ -142,6 +142,9 @@ add_library("${CLDNN_BUILD__PROJ}" STATIC
 set_property(TARGET "${CLDNN_BUILD__PROJ}" PROPERTY PROJECT_LABEL "${CLDNN_BUILD__PROJ_LABEL}")
 set_property(TARGET "${CLDNN_BUILD__PROJ}" PROPERTY OUTPUT_NAME   "${CLDNN_BUILD__PROJ_OUTPUT_NAME}")
 
+if(COMMAND add_cpplint_target)
```

Comment:
Maybe it makes sence to add `COMMAND add_cpplint_target` only for Ninja generator and don't change anything for other cases? Do we have a simple solution for this via cmake?

Reply:
I wouldn't want to add any generator specific logic for this simple change, especially when it doesn't lead to any visible improvements